lib/astarte_vmq_plugin.ex

```diff
@@ -255,4 +263,11 @@ defmodule Astarte.VMQ.Plugin do
 
     "#{realm_trunc}-#{device_id_trunc}-#{timestamp_hex_str}-#{rnd}"
   end
+
+  defp get_connection_reconciler_process!(client_id) do
```

Comment:
Let's be consistent with the naming, `connection_reconciler` -> `connection_synchronizer`